src/app/favourites/_components/FavouritesEmpty.tsx

```diff
@@ -0,0 +1,38 @@
+import Button from '@/components/ui/Button'
+import Image from 'next/image'
+import search from '../../../../public/search_cart.png'
+import Link from 'next/link'
+import { useState } from 'react'
+import { AuthModal } from '@/components/modals/AuthModal'
+
+export default function FavouritesEmpty() {
+  const [showModal, setShowModal] = useState(false)
```

Comment:
Remove this line - do not write logic to open auth modal

Reply:
Fixed.